drivers/vulkan/rendering_context_driver_vulkan.cpp

```diff
@@ -106,7 +106,7 @@ const char *RenderingContextDriverVulkan::get_tracked_object_name(uint32_t p_typ
 
 	return vkTrackedObjectTypeNames[p_type_index];
 #else
-	return "VK_TRACK_DRIVER_* disabled at build time";
+	return "VK_TRACK_*_MEMORY disabled at build time";
```

Comment:
```suggestion
	return "VK_TRACK_*_MEMORY was disabled at build-time.";
```

Reply:
Is that wise? get_tracked_object_name() is supposed to return a short name, not a sentence.

When built with proper support, get_tracked_object_name() will return strings like: "INSTANCE", "PHYSICAL_DEVICE".

The reason it returns "VK_TRACK_*_MEMORY disabled at build time" rather than "UNKNOWN" or "ERROR" is just to give the user a more descriptive information of what went wrong.

Reply:
That makes sense, I agree.